### ISSUE TITLE
Documentation Update

### ISSUE THREAD

**Issue by jvjones** [hastexo/hastexo-xblock#128]
Hello,

First off, great project, looking forward to getting this fully working.

The documentation for this project could do with a slight update to include how to configure this in Studio on Open edX.

For the providers field, the provider must be supplied in the following format:
`[{"name": "provider_name"}]`

Just sharing this nugget as spent a few hours working that out.

**Comment by fghaas** [comment]
@jvjones Yes you are correct. `providers` is a list of dictionaries, and wherever Studio expects that, you must use JSON. Which is not exactly *great* UX, but it's what Studio expects.

If you define multiple providers and you want to specify different capacities for them, you'd use:

```json
[
  { "capacity": 25, 
    "name": "provider1" },
  { "capacity": 50, 
    "name": "provider2" }
]
```

Also, note that it's a JSON, not a Python list. Thus:

```json
  { "capacity": 25, 
    "name": "provider1" },
  { "capacity": 50, 
    "name": "provider2" },
]
```

... would actually cause an error, as a trailing comma is not allowed in JSON.
